study.md

```diff
@@ -19,38 +19,44 @@ Please write the code that would take the following integer `007` and return the
 string `"700"`
 
 ```ruby
-  # your answer here
+(007 * 100).to_s
```

Comment:
On Mon, Apr 11, 2016 at 9:49 AM, Jason Weeks <notifications@github.com>
wrote:

> this works i was looking for a 007.to_s.reverse!


That drops the leading zeros...

     --Nick